evap/staff/importers/json.py

```diff
@@ -141,7 +143,7 @@ def get_log(self) -> str:
         log += self._make_heading("Name Changes")
         log += self._make_total(len(self.name_changes))
         for name_change in self.name_changes:
-            log += f"- {name_change.old_first_name_given} {name_change.old_last_name} → {name_change.new_first_name_given} {name_change.new_last_name} (email: {name_change.email})\n"
+            log += f"- [{name_change.old_title}] {name_change.old_last_name}, {name_change.old_first_name_given} → [{name_change.new_title}] {name_change.new_last_name}, {name_change.new_first_name_given} (email: {name_change.email})\n"
```

Comment:
is it intended that most names will then be like `[] Doe, John`?

Reply:
no it's fine, just wanted to check so that we don't open "Log should not display [] if no title is set" after the next update :D